gordo_components/cli/workflow_generator.py

```diff
@@ -196,17 +189,17 @@ def workflow_generator_cli(**kwargs: dict):
         template = wg.load_workflow_template(workflow_template)
 
     # Clear output file
-    if kwargs["output_file"]:
-        open(kwargs["output_file"], "w").close()  # type: ignore
-    for i in range(0, len(config.machines), kwargs["split_workflows"]):  # type: ignore
+    if context["output_file"]:
+        open(context["output_file"], "w").close()  # type: ignore
```

Comment:
can this logic not be fixed with some fancy click-type on "output_file" maybe?

Reply:
since you are fixing everyhting anywho I mean ;-)

Reply:
Nha seems not really :-/

Reply:
Did it despite your skepticism, lemme know what you think.

Reply:
nvm.. i see now it wouldn't work. :sweat: